examples/stress_tests/many_animated_sprites.rs

```diff
@@ -37,6 +43,8 @@ fn setup(
     assets: Res<AssetServer>,
     mut texture_atlases: ResMut<Assets<TextureAtlas>>,
 ) {
+    warn!(include_str!("warning_string.txt"));
```

Comment:
Where is this getting imported from / why does it exist? Even reading the PR description I couldn't quite piece it together.

Reply:
This was added to other stress tests here: https://github.com/bevyengine/bevy/pull/5029

Reply:
That's what I was referring to in the third paragraph of the PR description ("to be more in line with other examples").